doc/source/schemas/sequenceAnnotationmethods.rst

```diff
@@ -0,0 +1,457 @@
+SequenceAnnotationMethods
```

Comment:
Again, I think we differ in our use of "annotation" and "feature". FWIW, I think of annotations as information attached to entities in toto, e.g., a set of GO terms associated with a specific sequence, and features as information attached to specific regions of sequences. In that usage, the domain and range of features are much narrower than that for annotations (which can basically be anything attached to annything).